.github/workflows/pythonpackage.yml

```diff
@@ -30,9 +30,10 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8 yapf
+        pip install ../google-datacatalog-rdbms-connector
+        pip install ../google-datacatalog-postgresql-connector
```

Comment:
`PostgreSQL` is a common dependence for `Redshift` and `Greenplum` connectors, since there is not impact by installing it for the other connectors I'm using the same build file.